garrysmod/lua/vgui/dimagebutton.lua

```diff
@@ -1,19 +1,21 @@
 
 local PANEL = {}
 AccessorFunc( PANEL, "m_bStretchToFit", "StretchToFit" )
+AccessorFunc( PANEL, "m_bDepressImage", "DepressImage" )
 
 function PANEL:Init()
 
 	self:SetPaintBackground( false )
 	self:SetDrawBorder( false )
 	self:SetStretchToFit( true )
+	self:SetDepressImage( true )
 
 	self:SetCursor( "hand" )
 	self.m_Image = vgui.Create( "DImage", self )
 
 	self:SetText( "" )
 
-	self:SetColor( color_white )
+	self:SetColor( Color( 255, 255, 255, 255 ) )

```

Comment:
Why not using `color_white` constant ?

Reply:
Ah didn't notice that - my local file was not up-to-date.